tests/bot/bot.py

```diff
@@ -92,3 +92,12 @@ def test_group_and_command(self):
         msg.content = '!test'
         commands = list(self.bot.get_commands_for_message(False, None, '!', msg))
         self.assertEqual(commands[0][0], self.bot._commands[0])
+
+        msg.content = '?test a'
+        commands = list(self.bot.get_commands_for_message(False, None, '!', msg, ['?', ';']))
```

Comment:
We can probably update the majority of these tests if we deprecate single-prefix.